tests/python/contrib/test_ethosn/test_split.py

```diff
@@ -37,8 +37,9 @@ def test_split(dtype):
     trials = [
         ((1, 16, 16, 32), (2, 7, 10), 2),
         ((1, 12, 8, 16), 3, 1),
-        ((1, 66), 11, 1),
     ]
+    if tei.get_ethosn_api_version() < 2111:
```

Comment:
nit: it would be better to seperate out version specific tests in future. So that it is easy to follow the failures if any.